build.sbt

```diff
@@ -279,11 +277,11 @@ lazy val noPublishSettings = Seq(
  * This returns the youngest jar we released that is compatible with
  * the current.
  */
-val unreleasedModules = Set[String]("hadoop-test") //releases 0.11
+val ignoredModules = Set[String]("benchmarks")
 
 def youngestForwardCompatible(subProj: String) =
   Some(subProj)
-    .filterNot(unreleasedModules.contains(_))
+    .filterNot(ignoredModules.contains(_))
     .map {
     s => "com.twitter" %% (s"scalding-$s") % "0.16.0"
```

Comment:
I thought a later version had been released.

We should write code to just compute the previous version if the current version does not end in 0